packages/trpc/server/routers/viewer/availability.tsx

```diff
@@ -54,76 +54,6 @@ export const availabilityRouter = router({
     .query(({ input }) => {
       return getUserAvailability(input);
     }),
-  defaultValues: authedProcedure.input(z.object({ scheduleId: z.number() })).query(async ({ ctx, input }) => {
```

Comment:
Is this not needed anymore?

Reply:
I'm not sure why I built this endpoint and didn't just extend get: - I revisited the code and I couldn't tell why I did this so I removed it.